loader/src/ptracer/monitor.cpp

```diff
@@ -438,7 +438,7 @@ static bool ensure_daemon_created(bool is_64bit) {
 }
 
 #define CHECK_DAEMON_EXIT(abi)                                                   \
-  if (status##abi.supported && pid == status64.daemon_pid) {                     \
+  if (status##abi.supported && pid == status##abi.daemon_pid) {                     \
```

Comment:
```suggestion
  if (status##abi.supported && pid == status##abi.daemon_pid) {                   \
```

It should be aligned (the suggestion might not make it aligned, but that's just to explain what to align)